build.xml

```diff
@@ -1211,10 +1215,12 @@
           <pathelement path="${java.class.path}"/>
           <pathelement location="${stress.build.classes}"/>
           <pathelement location="${fqltool.build.classes}"/>
+          <pathelement location="${sstableloader.build.classes}"/>
```

Comment:
This is a bit suboptimal, but because `sstableloader-test` makes use of this macro we need to include the classes here as well.  While not perfect it doesn't feel too problematic to have to define these here.